src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -107,7 +107,6 @@ internal MvcRazorHost(IChunkTreeCache chunkTreeCache, RazorPathNormalizer pathNo
 
                     // Can't use nameof because IHtmlHelper is (also) not accessible here.
                     MarkAsHtmlEncodedMethodName = HtmlHelperPropertyName + ".Raw",
-                    AddHtmlAttributeValuesMethodName = "AddHtmlAttributeValues",
```

Comment:
We are always explicit about the configuration of the methods. We should add the new values.